docs/api/ShallowWrapper/type.md

```diff
@@ -1,14 +1,15 @@
-# `.type() => String|Function|null`
+# `.type() => String | Function | null`
```

Comment:
I believe the jsdoc-based convention here is to not have spaces around the `|`, even if it's less readable that way.

Reply:
i'll put them back